Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100755
--- /dev/null	(date 1743487103850)
+++ b/.gitignore	(date 1743487103850)
@@ -0,0 +1,5 @@
+/.idea/php.xml
+/.idea/shelf/Uncommitted_changes_before_Update_at_1_4_2025,_4_52 pm_[Changes]/shelved.patch
+/.idea/shelf/Uncommitted_changes_before_Update_at_1_4_2025__4_52pm__Changes_.xml
+/.idea/vcs.xml
+/.idea/workspace.xml
